### PR TITLE
Lesson5-XMLHttp request

### PR DESCRIPTION
**Pull request by bramakrishna16**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+</head>
+<body>
+<h1>Welcome</h1>
+<h5>Enter your name and comments</h5>
+<form action="/hi">
+     Name:<br>
+      <input id='name' name='name'><br>
+     Comments:<br>
+      <input id='comments' name='comments'><br><br>
+<input id='submit' value='submit' type='submit'>
```
Comment (songz) [inline public/index.html]: indentation! `gg=G`

File: app.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const app=express();
+const fs=require('fs');
+
+const handle=(req,res)=>{
+  const output= req.query.name + ' ' + req.query.comments;
```
Comment (allopez7) [inline app.js]: instead of req.query.name+...., use `${req.query.name}: ${req.query.comments}`

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      <input id='comments' name='comments'><br><br>
+      <input id='submit' value='submit' type='submit'>
+</form>
+<script>
+  const onSubmit = document.getElementById("submit");
+    onSubmit.onClick=()=>{
+      const name=document.getElementById("name").value;
+      const comments=document.getElementById("comments").value;
+      const httpRequest= new XMLHttpRequest();
+      httpRequest.open("GET","/hi?name=" + name +"&comments"+ comments);
```
Comment (allopez7) [inline public/index.html]: you can also use ${name} here with the back tick.

File: app.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const app=express();
+const fs=require('fs');
+
+const handle=(req,res)=>{
+  fs.writeFile("/home/bharath/gitcode/garagescript/public/file.txt",`${req.query.name} ${req.query.comments}` + "\n");
```
Comment (songz) [inline app.js]: try putting "\n" inside your ` block

**Pull request merged**